profiles/mercury.raml

```diff
@@ -81,6 +82,24 @@ validations:
           return true;
         }
 
+  camelcase-template-parameters:
+    message: All template/URI params *MUST* be lowerCamelCase
+    targetClass: apiContract.Parameter
+    functionConstraint:
+      code: |
+        function(parameter) {
+          const re = /^[a-z]+([A-Z]?[a-z0-9]+)*$/;
+
+          // Parameters include query, path and headers so we need to check binding to filter
+          if (parameter['apiContract:binding'] &&
+              parameter['apiContract:binding'] == 'path' &&
+              !re.test(parameter['core:name'])
+          ) {
+            return false;
```

Comment:
Since there's no additional logic here, you could simplify by inverting the conditional and returning that directly.

Reply:
done